subprojects/packagefiles/raylib/examples/meson.build

```diff
@@ -0,0 +1,261 @@
+    'shaders/shaders_multi_sample2d',
+    'shaders/shaders_palette_switch',
+    'shaders/shaders_postprocessing',
+    'shaders/shaders_raymarching',
+    'shaders/shaders_shadowmap',
+    'shaders/shaders_shapes_textures',
+    'shaders/shaders_simple_mask',
+    'shaders/shaders_spotlight',
+    'shaders/shaders_texture_drawing',
+    'shaders/shaders_texture_outline',
+    'shaders/shaders_texture_tiling',
+    'shaders/shaders_texture_waves',
+    'shaders/shaders_write_depth',
+    'shaders/shaders_vertex_displacement',
+  ],
+  'audio': [
+    'audio/audio_mixed_processor',
+    'audio/audio_module_playing',
+    'audio/audio_music_stream',
+    'audio/audio_raw_stream',
+    'audio/audio_sound_loading',
+    'audio/audio_sound_multi',
+    'audio/audio_stream_effects',
+  ],
+  'others': [
+    'others/easings_testbed',
+    'others/embedded_files_loading',
+    'others/raylib_opengl_interop',
+    'others/raymath_vector_angle',
+    'others/rlgl_compute_shader',
+  ],
+}
+
+uses_raygui_incorrectly = [
+  'shapes/shapes_draw_circle_sector',
+  'shapes/shapes_draw_rectangle_rounded',
+  'shapes/shapes_draw_ring',
+  'shapes/shapes_splines_drawing',
+]
+
+example_compile_args = []
+
+if target_platform == 'GLFW' or target_platform == 'SDL' or target_platform == 'RGFW'
+  example_compile_args += '-DPLATFORM_DESKTOP'
+endif
+
+fs = import('fs')
+
+# Detect if compiling in an MSYS2-based MinGW/Clang environment
+host_is_msys2 = host_machine.system() == 'windows' and cc.get_argument_syntax() == 'gcc'
+
+foreach category, files : examples_list
+
+  example_inc_dirs = []
+
+  if category == 'others'
+    example_inc_dirs += include_directories('others/external/include')
+  endif
+
+
+  foreach file : files
+    basename = fs.name(file)
+
+    specific_compile_args = []
+
+    if host_machine.system() == 'windows' and not host_is_msys2
+      # detect wrong dllimport / dllexport declarations and therefore wrong build or public compile arguments
+      if file in uses_raygui_incorrectly
+        #some files use ragui, and that also uses the same defines (USE_LIBTYPE_SHARED and BUILD_LIBTYPE_SHARED) as raylib, so it clashes here, this needs to be removed, after upstream fixes this!
+        specific_compile_args += '/wd4273'
+      else
+        specific_compile_args += '/we4273'
+      endif
+    endif
+
+
+    # this example is not supported on windows
+    if file == 'core/core_loading_thread' and host_machine.system() == 'windows'
+      continue
+    endif
+
+    if file == 'others/raylib_opengl_interop'
+      #this is needed, as this example doesn't include glad correctly
+      specific_compile_args += '-DGLAD_GLAPI_EXPORT'
+      #this is needed, as the hard copy of glad.h inside the examples/external folderincorrectly uses 'WIN32' instead of '_WIN32' as detection for windows
+      if host_machine.system() == 'windows'
+        specific_compile_args += '-DWIN32'
+      endif
+    endif
+
+
+    executable(
```

Comment:
These aren't installed, aren't `override_find_program`med (and should not be), and aren't stored into a variable for use by the parent project, so is there a use case for building them at all?  The wrap could be simplified a lot by not building the examples.